newIDE/app/src/VariablesList/VariablesList.js

```diff
@@ -293,7 +293,8 @@ const VariablesList = ({ onComputeAllVariableNames, ...props }: Props) => {
   const _saveToHistory = () => {
     props.historyHandler
       ? props.historyHandler.saveToHistory()
-      : setHistory(saveToHistory(history, props.variablesContainer));
+      : // TODO: pass actionType as argument of saveToHistory
```

Comment:
is it still a todo? looks like you're doing it

Reply:
I will pass actionType as an optional argument so that it will be less misleading when someone works on that in the future